.drone/drone.yml

```diff
@@ -1463,6 +1463,7 @@ steps:
     NFPM_PASSPHRASE:
       from_secret: gpg_passphrase
     NFPM_SIGNING_KEY_FILE: /drone/src/private-key.key
+    PROMTAIL_JOURNAL_ENABLED: "true"
```

Comment:
could you confirm whether "true" here should be in string or boolean ("true" vs true like at line 1454)?

Reply:
Good call; let me check